auslib/web/admin/views/base.py

```diff
@@ -83,3 +83,14 @@ def delete(self, *args, **kwargs):
         self.log.debug("processing DELETE request to %s" % request.path)
         with dbo.begin() as trans:
             return self._delete(*args, transaction=trans, **kwargs)
+
+
+def serialize_signoff_requirements(requirements):
```

Comment:
Hooray! It's good to see this get centralized. There might be an issue with this function though, or maybe one of the callers. Did you see the test failures like:
```
E       -                          u'required_signoffs': {u'releng': 1},
E       ?                         --                      -
E       
E       +                         'required_signoffs': {'releng': 1, 'relman': 1},
E       ?                                                         +++++++++++++```

I haven't seen any changes to the test data, so I _think_ these are legitimate failures.

Reply:
I believe the test failures are fixed. The biggest changes had to do with adding `original_row` to all the scheduled changes endpoints.